packages/nextcloud/test/version_check_test.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:nextcloud/nextcloud.dart';
+import 'package:test/test.dart';
+import 'package:version/version.dart';
+
+void main() {
+  group('Version check', () {
+    test('Null versions', () {
+      final check = VersionCheck(
+        versions: null,
+        // Invalid constraints to avoid accidental validation
+        minimumVersion: Version(2, 0, 0),
+        maximumExclusiveVersion: Version(1, 0, 0),
+      );
+      expect(check.isSupported, isTrue);
+    });
+
+    test('Empty versions', () {
+      final check = VersionCheck(
+        versions: const [],
+        // Invalid constraints to avoid accidental validation
+        minimumVersion: Version(2, 0, 0),
+        maximumExclusiveVersion: Version(1, 0, 0),
+      );
+      expect(check.isSupported, isTrue);
+    });
+
+    test('Multiple versions', () {
+      final check = VersionCheck(
+        versions: [
+          Version(0, 9, 9),
+          Version(1, 5, 0),
+          Version(1, 6, 0),
+          Version(2, 0, 0),
+        ],
+        minimumVersion: Version(1, 0, 0),
+        maximumExclusiveVersion: Version(1, 9, 9),
+      );
+      expect(check.isSupported, isTrue);
+    });
```

Comment:
Can you explain me how multiple versions work?
This list contains versions that are unsupported by the constraints like `0.9.9` and `2.0.0`. Does this mean that `isSupported` is just checking whether the list contains supported versions?

I couldn't find any helpflul information in the doc comments.

Reply:
Yep it checks if at least one version is supported (used for news and notes). I'll add it to the doc comment